tests/distributions/test_zero_inflated.py

```diff
@@ -64,28 +72,38 @@ def test_zip_mean_variance(gate, rate):
 @pytest.mark.parametrize("probs", [0.1, 0.5, 0.9])
 def test_zinb_0_gate(total_count, probs):
     # if gate is 0 ZINB is NegativeBinomial
-    zinb_ = ZeroInflatedNegativeBinomial(
-        torch.zeros(1), total_count=torch.tensor(total_count), probs=torch.tensor(probs)
+    zinb1 = ZeroInflatedNegativeBinomial(
+        total_count=torch.tensor(total_count), gate=torch.zeros(1), probs=torch.tensor(probs)
+    )
+    zinb2 = ZeroInflatedNegativeBinomial(
+        total_count=torch.tensor(total_count), gate_logits=torch.tensor(-99.9), probs=torch.tensor(probs)
```

Comment:
Why not test with `gate_logits=torch.tensor(-math.inf)`? You test with `+math.inf` below? Shouldn't the clamping logic handle `-math.inf`?

Reply:
that nan's out